code/datums/components/stationloving.dm

```diff
@@ -16,13 +16,13 @@
 	src.allow_death = allow_death
 	check_in_bounds() // Just in case something is being created outside of station/centcom
 
-/datum/component/stationloving/InheritComponent(datum/component/stationloving/newc, original, list/arguments)
+/datum/component/stationloving/InheritComponent(datum/component/stationloving/newc, original, inform_admins, allow_death)
 	if (original)
-		if (istype(newc))
+		if (newc)
 			inform_admins = newc.inform_admins
 			allow_death = newc.allow_death
-		else if (LAZYLEN(arguments))
-			inform_admins = arguments[1]
+		else
+			inform_admins = inform_admins
```

Comment:
Arghhhhhh. put an underscore for the proc argument, or an `src.` to distinguish. _Just do something, **anything**_